scripts/docker/zeppelin-base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.3
+MAINTAINER Mahmoud Elgamal <mahmoudf.elgamal@gmail.com>
```

Comment:
From the top of my head 
 - there must be ASF Apache 2.0 license header at the beginning of every file
 - ASF projects usually discourage author annotations, could you please replace this with something like Apache Zeppelin Developers <dev@zeppelin.apache.org>?

Attributions are kept though JIRA\Git commit logs.